### DISCUSSION TITLE
what happened to `sort_albums_by_name`?

### DISCUSSION THREAD

**Discussion by seamusdemora** [cmus/cmus#1393] [General]
The title says it all I think... the option to sort albums by 'name' (instead of the default 'date') was defined [in this tutorial](https://www.mankier.com/1/cmus), but my ver of cmus says, `Error: no such option sort_albums_by_name` in response to `toggle` and `set`.

**Comment by gavtroy** [comment]
It was only added in 2.11 so you might not have it yet.

**Reply by seamusdemora** [reply]
Oh - you're right... I have 2.10  :(   It's Debian, so maybe I can pick up 2.11 from backports? 
    **==> [No - it's not available from backports]**

Still kind of surprising... the document I referenced appears to be dated from 2010!?!

**Reply by gavtroy** [reply]
It does still say 2010 in [Doc/cmus.txt](https://github.com/cmus/cmus/blob/master/Doc/cmus.txt). You're right, the date should be getting bumped each time something is added to the document.
